aptos-move/block-executor/src/errors.rs

```diff
@@ -17,14 +17,39 @@ pub enum IntentionalFallbackToSequential {
     /// We defensively check resource group serialization error in the commit phase.
     /// TODO: should trigger invariant violation in the transaction itself.
     ResourceGroupSerializationError(String),
+    /// Execution of a thread yields a non-recoverable error from the VM. It it happens
+    /// during parallel execution, an error will be logged and sequential fallback taken.
+    FatalVMError(String),
+}
+
+pub(crate) fn log_panic_or(error: &PanicOr<IntentionalFallbackToSequential>, context: &str) {
+    match error {
+        PanicOr::Or(e) => match e {
+            IntentionalFallbackToSequential::ModulePathReadWrite => {
+                debug!("[Execution]: Module read & written {}", context);
+            },
+            IntentionalFallbackToSequential::ResourceGroupSerializationError(msg) => {
+                error!(
```

Comment:
should this be alert! ? (and below ones as well)

Reply:
probably makes sense. The resource group error is alerting when it's created too, but being defensive with this makes sense to me.